### PR TITLE
Checking in notebook that calls APS to analyze financial document (10k).

### PR DESCRIPTION
**Pull request by AbhijitP-009**

### CODE REVIEW

**Review by karanataryn — COMMENTED** [review]
Would be better to clear all outputs, makes it easier to read

File: notebooks/financial-docs-10k-example.ipynb [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "partitioned_file = partition_file(f, aryn_api_key, extract_table_structure=True, use_ocr=True, selected_pages=[23])"
```
Comment (karanataryn) [inline notebooks/financial-docs-10k-example.ipynb:60]: Add a comment here detailing the parameters you use

File: notebooks/financial-docs-10k-example.ipynb [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "partitioned_file"
```
Comment (karanataryn) [inline notebooks/financial-docs-10k-example.ipynb:70]: Could be combined with the earlier cell

File: notebooks/financial-docs-10k-example.ipynb [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# display the dataframe\n",
+    "industry_geographic_breakdown"
```
Comment (karanataryn) [inline notebooks/financial-docs-10k-example.ipynb:116]: this and the previous cell could be combined with the cell pulling out the table

File: notebooks/financial-docs-10k-example.ipynb [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+   "execution_count": null,
+   "id": "7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#pull out the tables from the list of elements\n",
+    "tables = []\n",
+    "for panda in pandas:\n",
+    "    if panda[0]['type'] == 'table':\n",
+    "        tables.append(panda[1])"
```
Comment (HenryL27) [inline notebooks/financial-docs-10k-example.ipynb:94]: would write the for loop as 
```python
for elt, dataframe in pandas:
    if elt['type'] == 'table':
        tables.append(dataframe)
```
just makes it a little more clear what you're working with

File: notebooks/financial-docs-10k-example.ipynb [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "11",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
```
Comment (HenryL27) [inline notebooks/financial-docs-10k-example.ipynb:138]: Drop this cell?

File: notebooks/financial-docs-10k-example.ipynb [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# visit https://www.aryn.ai/cloud to get a key\n",
+    "aryn_api_key = os.environ['ARYN_API_KEY'] "
```
Comment (karanataryn) [inline notebooks/financial-docs-10k-example.ipynb:37]: I think it's standard to have this as `"YOUR_ARYN_API_KEY"`, would be worth rechecking this

**Review by HenryL27 — APPROVED** [review]
two nits

File: notebooks/financial-docs-10k-example.ipynb [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pandas = tables_to_pandas(partitioned_file)"
```
Comment (karanataryn) [inline notebooks/financial-docs-10k-example.ipynb:80]: Add a comment here stating you're converting to pd Dataframe?

**Review by karanataryn — COMMENTED** [review]
Few changes

File: notebooks/financial-docs-10k-example.ipynb (hunk 3 above)
Comment (AbhijitP-009) [reply notebooks/financial-docs-10k-example.ipynb:116]: For the purpose of the demo, we want to highlight once cell which shows the dataframe.

File: notebooks/financial-docs-10k-example.ipynb (hunk 7 above)
Comment (AbhijitP-009) [reply notebooks/financial-docs-10k-example.ipynb:80]: I think the method name is self explanatory.

File: notebooks/financial-docs-10k-example.ipynb (hunk 2 above)
Comment (AbhijitP-009) [reply notebooks/financial-docs-10k-example.ipynb:70]: Want a cell to highlight the JSON that's outputted.

File: notebooks/financial-docs-10k-example.ipynb (hunk 1 above)
Comment (AbhijitP-009) [reply notebooks/financial-docs-10k-example.ipynb:60]: Ack.

File: notebooks/financial-docs-10k-example.ipynb (hunk 6 above)
Comment (AbhijitP-009) [reply notebooks/financial-docs-10k-example.ipynb:37]: I think the assumption here is the customer would already have setup the environment variable as  "ARYN_API_KEY" and not as "YOUR_ARYN_API_KEY". I see the point you're making, but I think most customers won't be setting the env variable as "YOUR_ARYN_API_KEY".

File: notebooks/financial-docs-10k-example.ipynb (hunk 4 above)
Comment (AbhijitP-009) [reply notebooks/financial-docs-10k-example.ipynb:94]: Ack

File: notebooks/financial-docs-10k-example.ipynb (hunk 5 above)
Comment (AbhijitP-009) [reply notebooks/financial-docs-10k-example.ipynb:138]: Ack.

File: notebooks/financial-docs-10k-example.ipynb (hunk 6 above)
Comment (karanataryn) [reply notebooks/financial-docs-10k-example.ipynb:37]: Yes, but take a look at the other notebooks. It's been standardized differently

Comment (AbhijitP-009) [reply notebooks/financial-docs-10k-example.ipynb:37]: Oh i see, other notebooks are not pulling it from the environment and just setting it as "YOUR_ARYN_API_KEY" and expecting the customer to set it. I can discuss this with you offline.

**Review by karanataryn — APPROVED** [review]

**Pull request merged**